src/Bicep.Core.Samples/Files/InvalidLoadFunctions_CRLF/main.symbols.bicep

```diff
@@ -37,15 +37,15 @@ var textLoadWrongEncoding01 = loadTextContent('Assets/encoding-iso.txt', 'us-asc
 var textLoadWrongEncoding05 = loadTextContent('Assets/encoding-ascii.txt', 'iso-8859-1')
 //@[4:27) Variable textLoadWrongEncoding05. Type: '32 = \n33 = !\n34 = "\n35 = #\n36 = $\n37 = %\n38 = &\n39 = \'\n40 = (\n41 = )\n42 = *\n43 = +\n44 = ,\n45 = -\n46 = .\n47 = /\n48 = 0\n49 = 1\n50 = 2\n51 = 3\n52 = 4\n53 = 5\n54 = 6\n55 = 7\n56 = 8\n57 = 9\n58 = :\n59 = ;\n60 = <\n61 = =\n62 = >\n63 = ?\n64 = @\n65 = A\n66 = B\n67 = C\n68 = D\n69 = E\n70 = F\n71 = G\n72 = H\n73 = I\n74 = J\n75 = K\n76 = L\n77 = M\n78 = N\n79 = O\n80 = P\n81 = Q\n82 = R\n83 = S\n84 = T\n85 = U\n86 = V\n87 = W\n88 = X\n89 = Y\n90 = Z\n91 = [\n92 = \\\n93 = ]\n94 = ^\n95 = _\n96 = `\n97 = a\n98 = b\n99 = c\n100 = d\n101 = e\n102 = f\n103 = g\n104 = h\n105 = i\n106 = j\n107 = k\n108 = l\n109 = m\n110 = n\n111 = o\n112 = p\n113 = q\n114 = r\n115 = s\n116 = t\n117 = u\n118 = v\n119 = w\n120 = x\n121 = y\n122 = z\n123 = {\n124 = |\n125 = }\n126 = ~'. Declaration start char: 0, length: 88
 var textLoadWrongEncoding06 = loadTextContent('Assets/encoding-ascii.txt', 'utf-8')
 //@[4:27) Variable textLoadWrongEncoding06. Type: '32 = \n33 = !\n34 = "\n35 = #\n36 = $\n37 = %\n38 = &\n39 = \'\n40 = (\n41 = )\n42 = *\n43 = +\n44 = ,\n45 = -\n46 = .\n47 = /\n48 = 0\n49 = 1\n50 = 2\n51 = 3\n52 = 4\n53 = 5\n54 = 6\n55 = 7\n56 = 8\n57 = 9\n58 = :\n59 = ;\n60 = <\n61 = =\n62 = >\n63 = ?\n64 = @\n65 = A\n66 = B\n67 = C\n68 = D\n69 = E\n70 = F\n71 = G\n72 = H\n73 = I\n74 = J\n75 = K\n76 = L\n77 = M\n78 = N\n79 = O\n80 = P\n81 = Q\n82 = R\n83 = S\n84 = T\n85 = U\n86 = V\n87 = W\n88 = X\n89 = Y\n90 = Z\n91 = [\n92 = \\\n93 = ]\n94 = ^\n95 = _\n96 = `\n97 = a\n98 = b\n99 = c\n100 = d\n101 = e\n102 = f\n103 = g\n104 = h\n105 = i\n106 = j\n107 = k\n108 = l\n109 = m\n110 = n\n111 = o\n112 = p\n113 = q\n114 = r\n115 = s\n116 = t\n117 = u\n118 = v\n119 = w\n120 = x\n121 = y\n122 = z\n123 = {\n124 = |\n125 = }\n126 = ~'. Declaration start char: 0, length: 83
 var textLoadWrongEncoding07 = loadTextContent('Assets/encoding-ascii.txt', 'utf-16BE')
-//@[4:27) Variable textLoadWrongEncoding07. Type: '㌲‽ ㌳‽‡ਲ਼㐠㴠∊㌵‽‣ਲ਼㘠㴠␊㌷‽‥ਲ਼㠠㴠☊㌹‽‧਴〠㴠⠊㐱‽ ਴㈠㴠⨊㐳‽‫਴㐠㴠Ⰺ㐵‽‭਴㘠㴠⸊㐷‽ ਴㠠㴠《㐹‽‱ਵ〠㴠㈊㔱‽″ਵ㈠㴠㐊㔳‽‵ਵ㐠㴠㘊㔵‽‷ਵ㘠㴠㠊㔷‽‹ਵ㠠㴠㨊㔹‽※ਸ਼〠㴠㰊㘱‽‽ਸ਼㈠㴠㸊㘳‽‿ਸ਼㐠㴠䀊㘵‽⁁ਸ਼㘠㴠䈊㘷‽⁃ਸ਼㠠㴠䐊㘹‽⁅਷〠㴠䘊㜱‽⁇਷㈠㴠䠊㜳‽⁉਷㐠㴠䨊㜵‽⁋਷㘠㴠䰊㜷‽⁍਷㠠㴠上㜹‽⁏ਸ〠㴠倊㠱‽⁑ਸ㈠㴠刊㠳‽⁓ਸ㐠㴠吊㠵‽⁕ਸ㘠㴠嘊㠷‽⁗ਸ㠠㴠堊㠹‽⁙ਹ〠㴠娊㤱‽⁛ਹ㈠㴠尊㤳‽⁝ਹ㐠㴠帊㤵‽ ਹ㘠㴠怊㤷‽⁡ਹ㠠㴠戊㤹‽⁣਱〰‽⁤਱〱‽⁥਱〲‽⁦਱〳‽⁧਱〴‽⁨਱〵‽⁩਱〶‽⁪਱〷‽⁫਱〸‽⁬਱〹‽⁭਱㄰‽⁮਱ㄱ‽⁯਱ㄲ‽⁰਱ㄳ‽ⁱ਱ㄴ‽⁲਱ㄵ‽⁳਱ㄶ‽⁴਱ㄷ‽⁵਱ㄸ‽⁶਱ㄹ‽⁷਱㈰‽⁸਱㈱‽⁹਱㈲‽⁺਱㈳‽⁻਱㈴‽⁼਱㈵‽⁽਱㈶‽⁾'. Declaration start char: 0, length: 86
+

```

Comment:
I think this is another encoding issue.

Reply:
By updating .editorconfig, you have also option to introduce rules that later on as also consumed be IDE editors and cli's. Whitespace is only one of basic ones fixing readability, In these cases to make clear separation it is good practice to make separate commits doing fixing of current source. Applaying rules can be done by vs code, or pre-commit etc. So before any commit, rules can be forced/appealed.  Encoding is also other option in .editorconfig, but wanted to start from easy part.

Reply:
@jeskew in that case I can add second  `.editorconfig` under `src/Bicep.Core.Samples/Files/` to keep their original encoding in this folder ?

Reply:
The existing `.editorconfig` file should not have affected these files (since the `.bicep` extension doesn't match any section in that file).

Could this PR maybe be scoped down a bit, and instead just trim trailing whitespace from .cs, .ts, .tsx, .js, and .resx files? I'm not sure which tool was used to prepare this, but my preference would be to leave the JSON and Bicep test artifacts untouched.